src/Microsoft.OData.Client/Serialization/Serializer.cs

```diff
@@ -361,10 +361,11 @@ internal void WriteEntry(EntityDescriptor entityDescriptor, IEnumerable<LinkDesc
         /// <summary>
         /// Writes the feed entry
         /// </summary>
-        /// <param name="objects">A list of descriptors to be written</param>
+        /// <param name="descriptors">A list of descriptors to be written</param>
         /// <param name="bulkUpdateGraph">An instance of the <see cref="BulkUpdateGraph"/></param>
+        /// <param name="linkDescriptors">Descriptor links for some descriptors.</param>

```

Comment:
"some descriptors" - I think instead of this you mention which descriptors gets passed and why...